README.md

```diff
@@ -132,6 +132,18 @@ $ cargo run
 ```
 in this repository.
 
+> **alternative method:** install the tool locally
+>
+> ```shell
+> $ cargo install --locked tokio-console
```

Comment:
I see that the tokio-console README does not have the `--locked` flag for its installation instructions. I'd be happy to remove this, but thought I'd keep it for now in case reviewers wanted to go one way or another.

Reply:
If using `--locked` is generally considered preferable, this is fine with me --- we may want to update the `tokio-console` crate's README as well. I don't have a strong opinion here.

Reply:
Looking at the [cargo install docs](https://doc.rust-lang.org/cargo/commands/cargo-install.html)...

> By default, the `Cargo.lock` file that is included with the package will be ignored. This means that Cargo will recompute which versions of dependencies to use, possibly using newer versions that have been released since the package was published. The `--locked` flag can be used to force Cargo to use the packaged `Cargo.lock` file if it is available. This may be useful for ensuring reproducible builds, to use the exact same set of dependencies that were available when the package was published. It may also be useful if a newer version of a dependency is published that no longer builds on your system, or has other problems. The downside to using `--locked` is that you will not receive any fixes or updates to any dependency. 

I think it might be worth including the `--locked` flag to ensure the user experience matches what's expected. However, I see the benefit of getting security / performance / bug fixes. I could go either way too, but lean towards `--locked`. What are your thoughts?